project/Paradox.scala

```diff
@@ -37,14 +38,20 @@ object Paradox {
       "extref.github.base_url" -> (GitHub.url(version.value) + "/%s"), // for links to our sources
       "extref.samples.base_url" -> s"$pekkoBaseURL/docs/pekko-samples/current/%s",
       "pekko.doc.dns" -> s"$pekkoBaseURL",
-      "scaladoc.pekko.base_url" -> s"$pekkoBaseURL/api/pekko/current/org/apache",
+      "scaladoc.pekko.base_url" -> s"$pekkoBaseURL/api/pekko/${projectInfoVersion.value}/org/apache",
       "scaladoc.pekko.http.base_url" -> s"$pekkoBaseURL/api/pekko-http/current/org/apache",
+      "scaladoc.org.apache.pekko.base_url" -> s"$pekkoBaseURL/api/pekko/${projectInfoVersion.value}",
```

Comment:
This is necessary because `scaladoc.pekko.base_url` only works with the `@apidoc` directive because its only `@apidoc` directive that has a concept of root package, i.e. https://github.com/apache/incubator-pekko/blob/a0c6bffe5d6bc5f012f7258bc7d9c95c77cf6379/project/Paradox.scala#L120

This means that another entry needs to be added for `scaladoc.org.apache.pekko.base_url`.